plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -141,7 +156,31 @@ export const resolvers = ({
       args: GeneralArgs,
       _context = {}
     ): Observable<JobLink> {
-      return runJob(args.id).map(({ jobId }) => ({ jobId }));
+      if (args.id) {
+        return runJob(args.id).map(({ jobId }) => ({ jobId }));
```

Comment:
Generally I'd prefer an early return under test condition and not the main logic under a condition.